scripts/build_docs.py

```diff
@@ -3,21 +3,25 @@
 
 To use::
 
-  $ python documentation/build_docs.py
+  $ python scripts/build_docs.py
 """
 
 from importlib.machinery import SourceFileLoader
 import os
 from subprocess import Popen
 
-docs_dir = os.path.dirname(os.path.abspath(__file__))
-version_file = os.path.join(docs_dir, os.pardir, "agentos", "version.py")
+scripts_dir = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Do you think it's worth moving the creation of these common path variables to `shared.py`?

Reply:
sounds good!